### PR TITLE
form editor improvements

### PR DESCRIPTION
**Pull request by opqdonut**
for #2608

# Checklist for author

Remove items that aren't applicable, check items that are done.

## Reviewability
- [x] Link to issue
- [x] Consider adding screenshots for ease of review

## Documentation
- [x] Update changelog if necessary
  - covered by previous entry
- [ ] Update manual/ (if applicable)
  - once all visual changes are done

### CODE REVIEW

**Comment by opqdonut** [comment]
# Screenshots

## Form editor title

### Before

form-editor-before

### After

form-editor-after

## Labels

### Before

labels-form-before
labels-organization-before

### After

labels-form-after
labels-organization-after

**Review by Macroz — APPROVED** [review]

File: src/cljs/rems/administration/create_form.cljs [hunk 1]
```diff
@@ -761,14 +761,14 @@
      [flash-message/component :top]
      (if loading?
        [:div [spinner/big]]
-       [:div.container-fluid
+       [:<>
```
Comment (marharyta) [inline src/cljs/rems/administration/create_form.cljs:764]: If we put this here, we might wanna make sure ```validation-errors-summary``` has a row class

Comment (marharyta) [reply src/cljs/rems/administration/create_form.cljs:764]: I would personally add a ```.row``` class to ```validation-errors-summary```

File: src/clj/rems/css/styles.clj [hunk 2]
```diff
@@ -685,6 +685,7 @@
     ;; make fieldset legends look the same as normal labels
     [:legend {:font-size "inherit"}]]
    [:.application-field-label {:font-weight "bold"}]
+   [:.administration-field-label {:font-weight "bold"}]
```
Comment (marharyta) [inline src/clj/rems/css/styles.clj:688]: ```font-weight: 600``` would be better imho

Comment (opqdonut) [reply src/clj/rems/css/styles.clj:688]: Looks like we only use normal (400), bold (700) and 300 so far, and only import web fonts for these weights. Why 600 here?

**Pull request merged**